### ISSUE TITLE
Drag combo inside another combo after collapse issue

### ISSUE THREAD

**Issue by goseign** [antvis/G6#2623]
I just used the example from [combo layout](https://g6.antv.vision/en/examples/net/comboForce), i just added combo expand-collapse mode, the issue is when I collapse any combo inside parent, the combo will go far beyond of the view, and you can't even zoom into it.
Here is the [demo](https://codesandbox.io/s/green-surf-yngut)

**Comment by Yanyan-Wang** [comment]
What if you assign the `relayout` to be `false` for `collapse-expand-combo`:
```
{
    type: "collapse-expand-combo",
    relayout: false
  }
```

The collapse expand combo behavior will not trigger re-layout for current graph

**Comment by goseign** [comment]
Yeah, I know it works, but without this setting it should also work, that’s why I open the issue, any way not a higher priority issue though.

Thanks.

**Comment by Yanyan-Wang** [comment]
solved by v4.7.15
